build/NuSpecs/AppxManifest.xml

```diff
@@ -27,11 +27,19 @@
     <Extension Category="windows.activatableClass.inProcessServer">
       <InProcessServer>
         <Path>Microsoft.ProjectReunion.dll</Path>
-          <ActivatableClass ActivatableClassId="Microsoft.ProjectReunion.ActivationRegistrationManager" ThreadingModel="both" />
-          <ActivatableClass ActivatableClassId="Microsoft.ProjectReunion.AppLifecycle" ThreadingModel="both" />
-		  <ActivatableClass ActivatableClassId="Microsoft.ProjectReunion.PushNotificationChannel" ThreadingModel="both" />
-		  <ActivatableClass ActivatableClassId="Microsoft.ProjectReunion.PushNotificationCreateChannelResult" ThreadingModel="both" />
-		  <ActivatableClass ActivatableClassId="Microsoft.ProjectReunion.PushNotificationManager" ThreadingModel="both" />
+        <ActivatableClass ActivatableClassId="Microsoft.ApplicationModel.DynamicDependency.AddPackageDependencyOptions" ThreadingModel="both" />
```

Comment:
It looks like some of Scott's changes got pulled into this PR like these DynamicDependency classes. It will likely cause a compile break checked in as is. Can you isolate our code completely from his?

Reply:
General feedback. We should move just these changes to a seperate branch and see if we can compile it without the AppLifecycle dependency. Thats probably what we need to check into our topic branch first.